migrations/versions/0315_lower_api_rate_limit.py

```diff
@@ -0,0 +1,19 @@
+"""
+
+Revision ID: 0315_lower_api_rate_limit
+Revises: 0314_no_reply_template
+Create Date: 2021-01-08 16:13:25
+
+"""
+from alembic import op
+
+revision = '0315_lower_api_rate_limit'
+down_revision = '0314_no_reply_template'
+
+def upgrade():
+    op.execute("ALTER TABLE services ALTER rate_limit DROP DEFAULT")
+    op.execute("ALTER TABLE services_history ALTER rate_limit DROP DEFAULT")
+
+def downgrade():
+    op.execute("ALTER TABLE services ALTER rate_limit SET DEFAULT '1000'")
+    op.execute("ALTER TABLE services_history ALTER rate_limit SET DEFAULT '1000'")
```

Comment:
You'll need to change your upgrade and downgrade methods.

`upgrade` is called when applying the migration. For us we want to:
- alter the default value and changed it to 1000
- update rows where the value is currently 3000 and set it to 1000

`downgrade` is when we want rollback a database migration. For us we would need to:
- alter the default value and changed it to 3000
- update rows where the value is currently 1000 and set it to 3000

You need 3 SQL statements for each method